dev/TreeView/TreeView_themeresources.xaml

```diff
@@ -33,7 +33,8 @@
             <StaticResource x:Key="TreeViewItemBorderBrushSelectedPointerOver" ResourceKey="SubtleFillColorTransparentBrush" />
             <StaticResource x:Key="TreeViewItemBorderBrushSelectedPressed" ResourceKey="SubtleFillColorTransparentBrush" />
             <StaticResource x:Key="TreeViewItemBorderBrushSelectedDisabled" ResourceKey="SubtleFillColorTransparentBrush" />
-            
+
+            <StaticResource x:Key="TreeViewItemBorderBrushMultiSelectSelected" ResourceKey="SubtleFillColorTransparentBrush" />
```

Comment:
Can this resource name be simplified or shortened?

Reply:
@karenbtlai I don't think we can drop anything from here, as just 'selected' applies to a regular selected tree view item (different visual state). I agree this one is long but I wanted it to be descriptive, not confusing

Reply:
Let's request feedback from more people as once it's added it cannot be removed. I think we can shorten it to simply *MultiSelect as that implies it being selected. @StephenLPeters @gabbybilka thoughts?

Reply:
I don't think you can shorten that way, MultiSelect is talking about the state of the treeview, not the item.  You could maybe do TreeViewItemBorderBrushMultiSelected, but to me that feels a little bit hidden.